test/runner.js

```diff
@@ -315,6 +315,17 @@ describe('Runner', function(){
   });
 
   describe('allowUncaught', function() {
+    var immediately;
```

Comment:
We need it to catch the exception in the test method, instead of catching it on a global error handler.